modules/ROOT/pages/authentication-authorization/manage-users.adoc

```diff
@@ -449,15 +505,19 @@ Available users can be seen using `SHOW USERS`, which will produce a table of us
 | {check-mark}
 
 | roles
-| Roles granted to the user.
+| Native roles granted to the user using the `GRANT ROLE` command.
+
+This is not necessarily the set of roles that a user will receive in practice. The latter depends on DMBS configuration as well as the user's xref:authentication-authorization/auth-providers.adoc[Auth Providers].
+
+Examples of where a user would receive a different set of roles from that which appears in this column would include where they use external (e.g. LDAP or OIDC) auth, or where `native` is not listed in the xref:configuration/configuration-settings.adoc#config_dbms.security.authorization_providers[`dbms.security.authorization_providers`] configuration setting.
 
 Will return `null` in community edition.
 | LIST OF STRING
```

Comment:
I know this isn't on you but shouldn't these have been updated to the Cypher types? so `LIST<STRING>` instead of `LIST OF STRING`? (would need to go over the whole table to check all of them conforms)

Reply:
also applies to the SHOW CURRENT USER table

Reply:
@renetapopova This seems to also apply to at least the different SHOW PRIVILEGE column tables as well, so it might be a slightly bigger task to just go over all of them and update where needed?

Reply:
Hej @Hunterness, I am reviewing the PR now, but because it's enormous, affects many pages, and has hundreds of comments, I am doing it locally and might be looking only into some of the comments that you have discussed. I'll do this in one commit so you can see all my updates and comment on them if I have missed something.

Reply:
By the way, I assume you have finished the review from your side, as I've already started updating it locally.

Reply:
yeah I believe I was done (before I found this and the comment below as well when looking at another task XD)

Reply:
Ok, then, when I am done, I'll look at all the unresolved comments to see if I can address something.

Reply:
I think this should be addressed after we merge this one in a separate PR.